packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs'
+import path from 'path'
+
+import execa from 'execa'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+export const command = 'run side'
+export const description = 'Run server for api in production'
+export const builder = (yargs) => {
+  yargs
+    .positional('side', {
+      default: 'api',
+      description: 'Which server to start',
+      type: 'string',
+    })
+    .epilogue(
+      `Also see the ${terminalLink(
+        'Redwood CLI Reference',
+        'https://redwoodjs.com/reference/command-line-interface#run'
+      )}`
+    )
+}
+
+export const handler = async ({ side = 'api' }) => {
+  // Currently we only support api anyway
+  // Not sure if we will support web in the future
+  if (side === 'api') {
+    if (
+      !fs.existsSync(
+        path.join(getPaths().base, 'node_modules', '.bin', 'api-server')
+      )
+    ) {
+      console.error(c.error('RedwoodJS API server dependency not found'))
+      console.error(
+        'Please install @redwoodjs/api-server to use this command\n'
+      )
+
+      console.log(
+        `Run ${c.green('yarn workspace api add @redwoodjs/api-server')} \n`
+      )
+
+      process.exit(1)
+    }
+
+    await execa('yarn api-server', ['--functions', './dist/functions'], {
```

Comment:
From the looks of it in [api-server/../index.ts](https://github.com/redwoodjs/redwood/blob/main/packages/api-server/src/index.ts), it seems @peterp deprecated `api-server`. And now that I look at this file, it seems we're effectively going to be duplicating it here with this code.

Now that api-server is included in the CRWA template packages, it seems to me we should consolidate into the `redwood` commands. And other suggestions about consolidating and deprecating?

Reply:
Two things here:

1. I can switch it over to the newer api server (I just copied what we were discussing for render setup)
2. I'm going to try importing the handler for the new server, rather than call it through execa. That way 🤞we don't need to duplicate the options

Reply:
Turns out `--functions` isn't a valid param for api-server!?

Reply:
If you don't want to duplicate the options then I would suggest exporting them from the `api-server` as a named export; probably by doing something like:

`export const options = { port: { ... }, socket: { ... }, rootPath: { ... } }`

Reply:
@dac09 during the meeting, did you say you tried Peter's export recommendation above?

Reply:
I haven't, but the reason I don't think this'll quite work is because api-server index runs the http server the moment you import. But we can have the options as a separate file I suppose! I'll investigate further with dom tomorrow!

Reply:
kkk updated! Now it does a direct proxy to rw-api-server, although lacks the nicer error I had before!